regression/scripts/weights/eVTOL_Weights_Buildup_Regression.py

```diff
@@ -0,0 +1,352 @@
+from SUAVE.Methods.Propulsion import propeller_design
+from SUAVE.Methods.Power.Battery.Sizing import initialize_from_mass
+from SUAVE.Core import (
+    Data, Container, Units,
+)
+
+
+#-------------------------------------------------------------------------------
+# Define Base Vehicle
+#-------------------------------------------------------------------------------
+
+def vehicle_setup():
+    vehicle = SUAVE.Vehicle()
+    vehicle.tag = 'eVTOL Weight Test'
+    
+    #---------------------------------------------------------------------------
+    # Vehicle-Level Properties
+    #---------------------------------------------------------------------------
+    
+    vehicle.mass_properties.takeoff         = 840.  * Units.kg
+    vehicle.mass_properties.operating_empty = 640.  * Units.kg
+    vehicle.mass_properties.max_takeoff     = 1000. * Units.kg
+    
+    vehicle.reference_area                  = 12.
+    
+    #---------------------------------------------------------------------------
+    # Fuselage Properties Properties
+    #---------------------------------------------------------------------------
+    
+    fuselage = SUAVE.Components.Fuselages.Fuselage()
+    fuselage.tag = 'fuselage'
+    
+    fuselage.lengths.total      = 5.5
+    fuselage.width              = 2
+    fuselage.heights.maximum    = 1.5
+    
+    vehicle.append_component(fuselage)
+    
+    #---------------------------------------------------------------------------
+    # Energy Network Properties
+    #---------------------------------------------------------------------------
+    
+    net = Battery_Propeller()
+    net.voltage = 375
+    
+    esc = SUAVE.Components.Energy.Distributors.Electronic_Speed_Controller()
+    esc.efficiency  = 0.95
+    net.esc         = esc
+    
+    payload = SUAVE.Components.Energy.Peripherals.Payload()
+    payload.power_draw              = 0.
+    payload.mass_properties.mass    = 200 * Units.kg
+    net.payload                     = payload
+
+    avionics = SUAVE.Components.Energy.Peripherals.Avionics()
+    avionics.power_draw = 20.
+    net.avionics        = avionics
+    
+    bat = SUAVE.Components.Energy.Storages.Batteries.Constant_Mass.Lithium_Ion()
+    bat.mass_properties.mass    = 300. * Units.kg
+    bat.specific_energy         = 300. * Units.Wh/Units.kg
+    bat.resistance              = 0.006
+    bat.max_voltage             = 400
+    initialize_from_mass(bat,bat.mass_properties.mass)
+    net.battery                 = bat
+    
+    vehicle.append_component(net)
+    
+    #---------------------------------------------------------------------------
+    # Main Wing
+    #---------------------------------------------------------------------------
+    
+    wing = SUAVE.Components.Wings.Main_Wing()
+    wing.tag = 'main wing'
+    
+    wing.aspect_ratio               = 4.
+    wing.thickness_to_chord         = 0.15
+    wing.taper                      = 1.
+    
```

Comment:
Probably can remove all these spaces

Reply:
Do you mean the tabs used to make the assigned values line up? I try to make everything in a particular section line up for easy readability, so while there's some extra space for these three, the values line up with the other wing properties assigned below.